nextflow.config.example

```diff
@@ -54,6 +56,13 @@ params {
     //
     sra_download = 0
```

Comment:
Does sra_download get used for anything?

Reply:
That's from the aspera PR, it allows the user to choose between Aspera or the SRA toolkit to download SRA files.